setup.py

```diff
@@ -36,7 +36,7 @@
         "resolvelib>=0.8.0",
         "html5lib>=1.1",
         "CacheControl[filecache]>=0.12.10",
-        "cyclonedx-python-lib>=0.11.1,<1.0.0",
+        "cyclonedx-python-lib>=1.1.0,<2.0.0",
```

Comment:
```suggestion
        "cyclonedx-python-lib>=1.0.0,<2.0.0",
```

Is this right? I'm not seeing anything in 1.1.0 that we require here: https://github.com/CycloneDX/cyclonedx-python-lib/blob/main/CHANGELOG.md#v110-2022-01-13

Reply:
Yeah, that was just me pinning to the latest minor release. 1.0.0 should also be perfectly compatible, so I can reduce the constraint if you'd like.